dnsdist/conf/dnsdist.conf

```diff
@@ -18,4 +18,5 @@ newServer("10.16.2.2")
 -- do not expose zones under .internal, especially catalog.internal
 addAction("internal.", RCodeAction(DNSRCode.REFUSED))
 
--- TOOD add webserver here for Prometheus monitoring
+-- web server for metrics scraping
+webserver("10.16.4.10:8083", "we+ensure+security+via+network+segmentation", "")
```

Comment:
As to why a password is used (and not an API key, which would the third argument -- empty string disables it), see https://github.com/PowerDNS/pdns/issues/8797#issuecomment-584056727 and below.

Reply:
maybe we should put that reference in the code, in case this behavior changes or someone is wondering

Reply:
done